provider/cluster_rosa_classic_resource.go

```diff
@@ -347,6 +347,11 @@ func (t *ClusterRosaClassicResourceType) GetSchema(ctx context.Context) (result
 				Optional:   true,
 				Validators: proxyValidators(),
 			},
+			"default_ingress": {
+				Description: "default_ingress",
```

Comment:
Please add more informative description here 
something similar to the CLI: https://github.com/openshift/rosa/blob/b694d0272c9a0c77c564ee0987a5b7fad8b41ccc/cmd/create/ingress/cmd.go#L45